<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>649</width>
    <height>427</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>我的记事本</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/image/note.png</normaloff>:/image/note.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>1</number>
    </property>
    <property name="topMargin">
     <number>1</number>
    </property>
    <property name="rightMargin">
     <number>1</number>
    </property>
    <property name="bottomMargin">
     <number>1</number>
    </property>
    <item>
     <widget class="QPlainTextEdit" name="textedit"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>649</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_F">
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveas"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>设置(&amp;S)</string>
    </property>
    <widget class="QMenu" name="menu_2">
     <property name="title">
      <string>高亮</string>
     </property>
     <addaction name="actionC"/>
     <addaction name="actionPython"/>
     <addaction name="actionJavaScript"/>
     <addaction name="actionHTML"/>
    </widget>
    <addaction name="menu_2"/>
    <addaction name="actionbackcolor"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>查看(&amp;V)</string>
    </property>
    <addaction name="actionView_History"/>
    <addaction name="actionClear_History"/>
   </widget>
   <widget class="QMenu" name="menu_V">
    <property name="title">
     <string>编辑(&amp;V)</string>
    </property>
    <widget class="QMenu" name="menu_K">
     <property name="title">
      <string>书签(&amp;K)</string>
     </property>
     <addaction name="actionAddBookmark"/>
     <addaction name="actionShowBookmarks"/>
     <addaction name="actionDeleteBookmark"/>
     <addaction name="actionGoToBookmark"/>
    </widget>
    <addaction name="menu_K"/>
   </widget>
   <addaction name="menu_F"/>
   <addaction name="menu"/>
   <addaction name="menu_3"/>
   <addaction name="menu_V"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="actionSaveas"/>
   <addaction name="separator"/>
   <addaction name="actionbackcolor"/>
   <addaction name="actionView_History"/>
   <addaction name="actionClear_History"/>
   <addaction name="separator"/>
   <addaction name="actionAddBookmark"/>
   <addaction name="actionShowBookmarks"/>
   <addaction name="actionDeleteBookmark"/>
   <addaction name="actionGoToBookmark"/>
  </widget>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/image/new.png</normaloff>:/image/new.png</iconset>
   </property>
   <property name="text">
    <string>新建(&amp;N)</string>
   </property>
   <property name="toolTip">
    <string>新建</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionOpen">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/image/open.png</normaloff>:/image/open.png</iconset>
   </property>
   <property name="text">
    <string>打开(&amp;)</string>
   </property>
   <property name="toolTip">
    <string>打开</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/image/save.png</normaloff>:/image/save.png</iconset>
   </property>
   <property name="text">
    <string>保存(&amp;S)</string>
   </property>
   <property name="toolTip">
    <string>保存</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionSaveas">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/image/saveas.png</normaloff>:/image/saveas.png</iconset>
   </property>
   <property name="text">
    <string>另存为(&amp;A)</string>
   </property>
   <property name="toolTip">
    <string>另存为</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F6</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>退出</string>
   </property>
   <property name="toolTip">
    <string>退出</string>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionC">
   <property name="text">
    <string>C</string>
   </property>
  </action>
  <action name="actionPython">
   <property name="text">
    <string>Python</string>
   </property>
  </action>
  <action name="actionJavaScript">
   <property name="text">
    <string>JavaScript</string>
   </property>
  </action>
  <action name="actionHTML">
   <property name="text">
    <string>HTML</string>
   </property>
  </action>
  <action name="actionbackcolor">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/image/backcolor.png</normaloff>:/image/backcolor.png</iconset>
   </property>
   <property name="text">
    <string>编辑器背景色</string>
   </property>
   <property name="toolTip">
    <string>编辑器背景色</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionView_History">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/image/view_history.png</normaloff>:/image/view_history.png</iconset>
   </property>
   <property name="text">
    <string>View_History</string>
   </property>
   <property name="toolTip">
    <string>查看历史</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionClear_History">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/image/delete_history.png</normaloff>:/image/delete_history.png</iconset>
   </property>
   <property name="text">
    <string>Clear_History</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionAddBookmark">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/image/add.png</normaloff>:/image/add.png</iconset>
   </property>
   <property name="text">
    <string>增加</string>
   </property>
  </action>
  <action name="actionShowBookmarks">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/image/show.png</normaloff>:/image/show.png</iconset>
   </property>
   <property name="text">
    <string>显示</string>
   </property>
  </action>
  <action name="actionDeleteBookmark">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/image/delete.png</normaloff>:/image/delete.png</iconset>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </action>
  <action name="actionGoToBookmark">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/image/goto.png</normaloff>:/image/goto.png</iconset>
   </property>
   <property name="text">
    <string>跳转</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
